app/main.py

```diff
@@ -0,0 +1,8 @@
+from fastapi import FastAPI
+from app.api.v1.endpoints import health
+
+# Initialize FastAPI app
+app = FastAPI()
+
+# Include routers
+app.include_router(health.router, prefix="/api/v1", tags=["health"])
```

Comment:
for health api.. don't need to have any prefix.. it can just be localhost:{port}/health